GoogleForCreators/web-stories-wp
# Redesign: Excerpt panel

## Context
Updates design for the Excerpt panel.


## Summary



## Relevant Technical Choices



## To-do



## User-facing changes
| Before | After |
|--------|------|
| ![Screenshot 2021-03-17 at 09 59 38](https://user-images.githubusercontent.com/3294597/111433886-25916480-86ff-11eb-82cb-391714cc5673.png) | ![Screenshot 2021-03-17 at 09 59 09](https://user-images.githubusercontent.com/3294597/111433905-2c1fdc00-86ff-11eb-8afb-f5ae43a9115e.png) |


## Testing Instructions
1. Open the Document panel
2. Verify the Excerpt panel looks as in [Figma](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=3897%3A63826)


### QA






This PR can be tested by following these steps:

1. Open the Document panel
2. Verify the Excerpt panel looks as in [Figma](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=3897%3A63826)
### UAT







This PR can be tested by following these steps:

1.
## Checklist




---



Fixes #6515
